pxr/imaging/hd/meshTessellations.cpp

```diff
@@ -0,0 +1,145 @@
+// TODO
+#include "pxr/imaging/hd/meshTessellations.h"
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+// Find if an edge(u, v) is found in edges.
+bool IsEdge(
+    const std::map<std::pair<int, int>, int>& edges, 
+    const int u, 
+    const int v
+)
+{
+    const auto edge = (u < v) ? std::make_pair(u, v) : std::make_pair(v, u);
+    const auto it = edges.find(edge);
+    if (it == edges.end()) {
+        return false;
+    }
+    return it->second == 1;
+}
+
+// Given a triangle, returns its edge flag.
+// edgeFlag is used for inner-line removal of non-triangle
+// faces on wireframe shading.
+//
+//          0__                0  0   0__
+//        _/|\ \_            _/.  ..   . \_
+//      _/  | \  \_   ->   _/  .  . .   .  \_
+//     /  A |C \ B \_     /  A .  .C .   . B \_
+//    1-----2---3----4   1-----2  1---2   1----2
+//
+//  Type   EdgeFlag    Draw
+//    -       0        show all edges
+//    A       1        hide [2-0]
+//    B       2        hide [0-1]
+//    C       3        hide [0-1] and [2-0]
+int EdgeFlag(
```

Comment:
These helper functions either need to be tagged as `static` or put in an anonymous namespace.  I'd recommend anonymous namespace.

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rs-unnamed2

It would be common to prefix these with `_`.